python/ray/train/data_parallel_trainer.py

```diff
@@ -444,7 +443,7 @@ def training_loop(self) -> None:
             driver_ip=ray.util.get_node_ip_address(),
             driver_node_id=ray.get_runtime_context().get_node_id(),
             experiment_name=session.get_experiment_name(),
-            run_id=uuid.uuid4().hex,
+            run_id=session.get_run_id(),
```

Comment:
Just for my own understanding, this `training_loop` function is part of `tuner.fit()` function that will be called in a tune process. Therefore, the `session.get_run_id()` will actually get the `run_id` from a tune process, although this `seesion` is imported from `ray.train._internal`. This session is actually initialized inside `functional_trainable.py` which defined under `tune/trainable/function_trainable.py`. cc @justinvyu @matthewdeng

Reply:
That's right, the `training_loop` here is the Train driver logic that is running inside the `tune.FunctionTrainable`. So the "session" refers to the Ray Tune session, not the Ray Train Worker session.